### ISSUE TITLE
Any plans to update this?

### ISSUE THREAD

**Issue by genyded** [praneybehl/meteor-react-select#3]
With Meteor 1.3 we need react-select 1.0.0-beta9 or higher for it to work. Do you plan to update it?

**Comment by praneybehl** [comment]
Mate with Meteor 1.3 there is full support for NPM and all you need is to add it straight from the npm module using npm install react-select

It doesn't makes any sense for me to update since meteor packages system will no longer be maintained. Hope is helps.

**Comment by genyded** [comment]
Yes but it has pre-compiled css modules and causes a problem if you use any other component and want to compile their css modules. See [this issue](https://github.com/nathantreid/meteor-css-modules/issues/33)
